sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceClientBuilder.java

```diff
@@ -114,7 +114,7 @@ public BlobServiceAsyncClient buildAsyncClient() {
             policies.add(new SASTokenCredentialPolicy(sasTokenCredential));
         }
         HttpPolicyProviders.addBeforeRetryPolicies(policies);
-        policies.add(new RequestRetryPolicy(retryOptions));
+       // policies.add(new RequestRetryPolicy(retryOptions));
```

Comment:
Did this mean to be removed? What does the retry policy do if the error is `UnexpectedLengthException`?

Reply:
No. Just for testing. Will uncomment.